user_templates/variables.cfg

```diff
@@ -37,6 +37,9 @@ variable_zendstop_position: -1, -1 # change it ONLY if not using the auto z cali
 ## Force always a full homing and QGL/Z_TILT during the START_PRINT macro
 variable_force_homing_in_start_print: False
 
+## Always force an adaptive bed mesh during the START_PRINT macro
+variable_force_mesh: False
+
```

Comment:
Remove this as it's not needed for this and would add more confusion since it would need a restart of Klipper each time you want to change it and it's not really flexible to configure it per print